systemtest/approvals/TestServiceDestinationAggregation.approved.json

```diff
@@ -17,7 +17,8 @@
             "event": {
                 "agent_id_status": "missing",
                 "ingested": "dynamic",
-                "outcome": "success"
+                "outcome": "success",
+                "received": "dynamic"
```

Comment:
Why is event.received still present? Isn't apm-data updated to remove the field?